js/framework/articleHTMLRenderer.js

```diff
@@ -53,8 +54,9 @@ var ArticleHTMLRenderer = new Knowledge.Class({
     _get_html: function (model) {
         let engine = DModel.Engine.get_default();
         let domain = engine.get_domain();
-        let [, bytes] = domain.read_uri(model.id);
-        return bytes.get_data().toString();
+        let [, data_gbytes] = domain.read_uri(model.id);
+        let data_uint8array = ByteArray.fromGBytes(data_gbytes);
+        return ByteArray.toString(data_uint8array);
```

Comment:
Thanks for checking! That one generates a warning in the newer version of gjs here :)

> Some code called array.toString() on a Uint8Array instance. Previously this would have interpreted the bytes of the array as a string, but that is nonstandard. In the future this will return the bytes as comma-separated digits. For the time being, the old behavior has been preserved, but please fix your code anyway to explicitly call ByteArray.toString(array).

The idea is that warning gets shown when we call toString on a Uint8Array instance, while in previous versions of gjs we were working with a ByteArray instance.